packages/domain-picker/src/domain-categories/index.tsx

```diff
@@ -35,6 +29,10 @@ const DomainPickerCategories: React.FunctionComponent< Props > = ( {
 		onSelect( slug );
 	};
 
+	const domainCategories = useSelect( ( select ) =>
+		select( 'automattic/domains/suggestions' ).getCategories()
```

Comment:
Shouldn't we follow the pattern to register the store to get the key and then use to key to select data from the store? Like here: https://github.com/Automattic/wp-calypso/pull/43045/files#diff-0d151bc68b92f69ebc364c9a4b674116R9-R11

Reply:
That's a problem with the domains suggestions store because it requires configuration on registration. A shared component shouldn't need to concern itself with application configuration.

Expecting the store to be registered by the application the store correctly and using the string key directly seems to be the best.

We could consider some defensive practices, like throwing a helpful error if the store is required but not found:

```
Error: The domain-picker component expected `automattic/domains/stores` to be registered but it was not found. Please register the store before this component is used. See some-link-to-documentation.
```